src/pages/Lend/LendHeader.vue

```diff
@@ -49,6 +50,19 @@ export default {
 			default: false,
 		},
 	},
+	serverCacheKey: props => {
+		const {
+			browseUrl,
+			filterUrl,
+			sidePinnedFilterPadding,
+			hardLeftAlign,
+			sideArrowsPadding
+		} = props;
+		const getValue = value => {
+			return value ? `-${value}` : '';
+		};
+		getCacheKey(`${filterUrl}${getValue(browseUrl)}${getValue(sidePinnedFilterPadding)}${getValue(hardLeftAlign)}${getValue(sideArrowsPadding)}`); // eslint-disable-line max-len
```

Comment:
It seems like the filterUrl + browserUrl shouldn't be used on this component. Rather they would go on the ViewToggle where they would actually differ? Or, does it need to be the parent (this component) that manages that...

I just wonder if either `filterUrl` or `browserUrl` will contain query params and result in a lot of instances of this being cached...

Reply:
@mcstover this is a result is that is currently being generated there **_/lend/filter-/lend-by-category-true_**

Reply:
Ok suppose that's ok. The "/" characters are a little odd but no query params in site!